public/video-ui/src/components/Create.tsx

```diff
@@ -0,0 +1,173 @@
+        <div className="create-form__option-controls">
+          <div className="create-form__option-radio-label-and-icon">
+            {this.iconMap[videoCreateOptionDetails.id]}
+            <label
+              htmlFor={videoCreateOptionDetails.id}
+              className="create-form__option-radio-label"
+            >
+              {videoCreateOptionDetails.title}
+            </label>
+          </div>
+          <input
+            type="radio"
+            className="create-form__option-radio"
+            id={videoCreateOptionDetails.id}
+            name="videoPlayerFormat"
+            value={videoCreateOptionDetails.id}
+            checked={isSelected}
+            onChange={() => this.setState({ videoCreateOption: videoCreateOptionDetails.id })}
+          />
+        </div>
+        {
+          <div className={
+            "create-form__option-specifications " +
+            (isSelected ? 'create-form__option-specifications--visible' : '')
+          }>
+            <ul aria-label="positives">
+              {videoCreateOptionDetails.specifications.positive.map(positiveSpecification => (
+                <li key={positiveSpecification} className="create-form__list-item--specification">
+                  {positiveSpecification}
+                  <Checkmark/>
+                </li>
+              ))}
+            </ul>
+            <ul aria-label="negatives">
+              {videoCreateOptionDetails.specifications.negative.map(negativeSpecifiation => (
+                <li key={negativeSpecifiation} className="create-form__list-item--specification">
+                  {negativeSpecifiation}
+                  <Cross/>
+                </li>
+              ))}
+            </ul>
+            <ul aria-label="other information">
+              {videoCreateOptionDetails.specifications.info.map(infoSpecification => (
+                <li key={infoSpecification} className="create-form__list-item--specification">
+                  {infoSpecification}
+                  <Info/>
+                </li>
+              ))}
+            </ul>
+          </div>
+        }
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <div className={'create-form ' + (this.props.inModal ? 'create-form--modal' : '')}>
+        <h2 className="create-form__heading--level-2">
+          Create New Video
+        </h2>
+        <div className="create-form__contents">
+          <div className="create-form__working-title">
+            <label className="create-form__label" htmlFor="working-title">Working Title</label>
+            <input
+              id="working-title"
+              name="Working Title"
+              onChange={(event) => this.setState({ workingTitle: event.target.value })}
+              value={this.state.workingTitle}
+              className="form__field"
+            />
+          </div>
+          <div>
+            <h4 className="create-form__heading--level-3">
+              Video Player Format
+            </h4>
```

Comment:
Given the h2 above, I’d expect a h3 rather than a h4 (to avoid skipping heading levels): what’s the reason for the h4?